apps/channels/models.py

```diff
@@ -34,18 +37,30 @@ def form(self):
         return ChannelForm(initial={"platform": self})
 
     def extra_form(self, *args, **kwargs):
-        from apps.channels.forms import TelegramChannelForm, WhatsappChannelForm
+        from apps.channels import forms
 
         match self:
             case self.TELEGRAM:
-                return TelegramChannelForm(*args, **kwargs)
+                return forms.TelegramChannelForm(*args, **kwargs)
             case self.WHATSAPP:
-                return WhatsappChannelForm(*args, **kwargs)
+                return forms.WhatsappChannelForm(*args, **kwargs)
+            case self.FACEBOOK:
+                initial = kwargs.get("initial", {})
+                initial.setdefault("verify_token", str(uuid.uuid4()))
+                kwargs["initial"] = initial
+                return forms.FacebookChannelForm(*args, **kwargs)
+
+
+class ExperimentChannelObjectManager(models.Manager):
+    def filter_extras(self, key: str, value: str, platform: ChannelPlatform, team: str):
```

Comment:
We can keep the platform and team separate from this method call. I'm in two minds. I think we can change it later if need be

Reply:
nit: if team and platform and required I'd put them first in the args list. Otherwise make them optional.

I'm having a hard time articulating why I think that's better. Perhaps because they are like constant filters and the key / value are the dynamic ones? I guess it's also analogous to placing the big rocks first: team -> platform -> key+value

Reply:
https://github.com/dimagi/open-chat-studio/pull/115/commits/ce2298bedab43b56ec9d9aeb6e4fb6a1a217534e